README.md

```diff
@@ -16,7 +16,7 @@
 </div>
 
 <div align="center">
-  <img alt="Wukong CLI" src="./.github/img/wukong-help.png" >
+  <img alt="Wukong CLI" src="./.github/img/wukong_logo.png" width="200" height="200" >
```

Comment:
it is better to show the logo on the top, and still show the cli help screenshot here